dts/bindings/can/can.yaml

```diff
@@ -28,6 +28,21 @@ properties:
       category: required
       description: bus spees in Baud/s
       generation: define
+    sjw:
+      type: int
+      category: required
+      description: Resynchronization jump width (ISO 11898-1)
+      generation: define
+    prop_seg_phase_seg1:
```

Comment:
Why is this prop seg + phase, but the next property is just phase?

Reply:
------------------------------------------------------------------------
|  SYNC    | PROP_SEG | PHASE_SEG1 | PHASE_SEG2 |
------------------------------------------------------------------------
|Always 1 |     prop_seg_phase_seg1     |   phase_seg2  |
------------------------------------------------------------------------
^                                                             ^
Start                                                Sample point
PROP_SEG is the signal propagation delay
PHASE_SEG1 and PHASE_SEG2 define the sample point.
Because PROP_SEG and PHASE_SEG1 + 1 is the time before sampling it makes no sense
to have individual parameters for that.

"In a CAN implementation, Prop_Seg and Phase_Seg1 do not need to be programmable separately; it is sufficient to program the sum of Prop_Seg and Phase_Seg1." [ISO 11898-1]